### PR TITLE
Script code

### LINKED ISSUE #135: ScriptCode localization support
I need to support Locale for Serbian language that has both Cyrilic and Latin scriptCode, and therefore locales look like this: 

Locale.fromSubtags(languageCode: 'sr', scriptCode: 'Cyrl', countryCode: 'RS')
Locale.fromSubtags(languageCode: 'sr', scriptCode: 'Latn', countryCode: 'RS')

And due to this requirement I have 2 JSON files for Serbian: 
1. sr_Cyrl-RS.json
2. sr_Latn-RS.json

But unfortunately plugin just gives me error "Unable to load asset "lang/sr-RS.json"
How can I include support for scriptCode into my locale using this plugin?

App Store just released support for several contries including Serbia, so I assume a lot of iOS developers will now start supporting this language and will need this plugin to do so.

### PR DESCRIPTION
**Pull request by Overman775**
- add support sciptCode, fix #135 
- fix useOnlyLangCode in assetLoader
- refactoring, create utils.dart
- updare reamde and example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,10 +111,12 @@ import 'package:easy_localization/easy_localization.dart';
 void main(){
   runApp(EasyLocalization(
     child: MyApp(),
-    supportedLocales: [Locale('en', 'US'), Locale('ar', 'DZ')],
+    supportedLocales: [Locale('en', 'US'), Locale('ar', 'DZ')], // [Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant', countryCode: 'HK')]
```
Comment (aissat) [inline README.md:114]: if possible found other way todo this without ` Locale.fromSubtags`

Comment (Overman775) [reply README.md:114]: nope... only use `fromSubtags`, locale conatain olnly two parameters   `this._languageCode this._countryCode`

https://api.flutter.dev/flutter/dart-ui/Locale/Locale.html

Comment (Overman775) [reply README.md:114]: Or create new Locale class and oweride... i think its bad way

File: lib/src/bloc/easy_localization_bloc.dart [hunk 2]
```diff
@@ -6,12 +6,16 @@ class Resource {
...
   Translations _translations;
-  Translations get translations => _translations;
-
+  Translations get translations => _translations; 
   Resource({this.locale, this.assetLoader, this.path, this.useOnlyLangCode});
 
   void loadTranslations() async {
-    var data = await assetLoader.load(path, locale);
+    Map<String,dynamic> data;
+    if (useOnlyLangCode){
```
Comment (Overman775) [inline lib/src/bloc/easy_localization_bloc.dart:14]: need flutter test witch diffrent aseets (en, en_US) for compare expect

**Pull request merged**